src/components/entity/ha-chart-base.html

```diff
@@ -258,11 +265,25 @@
         hidden: chart.isDatasetVisible(i)
       })));
       this.set('unit', this.data.unit);
-    }
+    },
+    _formatTicks(ticks) {
+      return ticks.map((tick) => {
+        window.hassUtil.formatTime(tick.value);
```

Comment:
You need to return the value. If you omit the curly braces around the lambda body, it will have an implicit return:

```js
return ticks.map(tick => window.hassUtil.formatTIme(tick.value));
```